grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy

```diff
@@ -223,6 +258,8 @@ class BioActivityController {
         }
         model.activities = results?.activities
         model.total = results?.total
+        String userId = userService.getCurrentUserId()
+        model.showCrud = userId ? projectService.isUserAdminForProject(userId, id) : false
```

Comment:
Does this mean we only show the add/edit/delete options for project admins? Should also be for the owner of the activity/record, shouldn't it?